src/components/Header/CustomHeader.tsx

```diff
@@ -29,6 +30,16 @@ function CustomHeader() {
         }
     }, [location.pathname])
 
+    const [name, setName] = useState('')
+
+    const token = useSelector((state: RootState) => state.auth.authtoken as string)
+
+    useEffect(() => {
```

Comment:
das kann in das andere useEffect mit rein oder nicht?

Reply:
ist jetzt angepasst :)